src/components/connection_handler/src/connection_handler_impl.cc

```diff
@@ -285,17 +285,27 @@ uint32_t ConnectionHandlerImpl::OnSessionStartedCallback(
     const uint8_t session_id,
     const protocol_handler::ServiceType& service_type,
     const bool is_protected,
-    uint32_t* hash_id) {
+    uint32_t* hash_id,
+    bool* out_start_protected) {
```

Comment:
@istoilovgithub, do not forget to add `out_` prefix to `hash_id`. And according our coding stile we mostly use references for output parameters, spatially for a `POD`